config/endpoint/ssh/ssh.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 
 // Validate the SSH configuration
 func (cfg *Config) Validate() error {
+	if len(cfg.Username) == 0 && len(cfg.Password) == 0 {
+		return nil
```

Comment:
Can you add a comment above this line: `// If there's no username and password, this endpoint can still check the SSH banner, so the endpoint is still valid`